packages/foam-core/src/initializeNoteGraph.ts

```diff
@@ -0,0 +1,30 @@
+
+import glob from 'glob';
+import { promisify } from 'util';
+import fs from 'fs';
+import os from 'os';
+import detectNewline from 'detect-newline';
+import { NoteGraph } from './note-graph';
+import { createNoteFromMarkdown } from './markdown-provider';
+
+const findAllFiles = promisify(glob);
+
+export const initializeNoteGraph = async (aboluteDir: string) => {
```

Comment:
Typo: Rename variable to `absoluteDir`

Reply:
Renamed it to `workspacaPath` since that made more sense than` absoluteDir`.